portafly/doc/ux_specs/index.md

```diff
@@ -8,6 +8,7 @@ Please refer to the single page files together with mockups linked in each page
 ### Global components
 * [Toolbar](./global_components/toolbar.md) -- the toolbar is a collection of UI components that allow a user to manage and manipulate a data set. A toolbar will almost always be present in pages that include a table, data list etc.
 * [Modal](./global_components/modal.md) -- the modal is an overlay dialog triggered to allow users taking contextual bulk actions or confirming destructive actions.
+* [Alerts](./global_components/alerts.md) -- Alerts are used to notify the user about a change in system status or other event that needs attention.
```

Comment:
Are alerts a mean to notify only, or also to prompt users to perform an action? 
Examples:
* Notify that the number of hits to an endpoint has reached its limit (no action required)
* To create an application, an application plan is required. Inform the user about this, and prompt the creation of an application plan (action required)

Perhaps we should distinguish between:
* Notifications: informative messages
* Alerts: informative messages requiring action from the user.

Reply:
I think `Alert` refers to a [Patternfly component](https://www.patternfly.org/v4/documentation/react/components/alert) and yes it is possible to include a prompt for the user. They call it `AlertAction`.

I personally would keep it simple and use _Alert_. For informative messages requiring action I would use _Alert with an action_.

Reply:
@porueesq thank you for your comment. I agree with @josemigallas, and for the record, I have used `Alert` because that is how the component is called in PF, where there is no differentiation between an 'alert' and a 'notification'